scripts/xe-backup-metadata

```diff
@@ -134,11 +156,25 @@ function cleanup {
    fi
```

Comment:
We should run `ShellCheck` in the CI, there are about 66 warnings reported on this file, which means that about 20% of this script is slightly wrong (pre-existing problem, not necessarily with the changes here). I'll attempt to paste the relevant output from shellcheck as comments.